requirements.txt

```diff
@@ -2,7 +2,7 @@ requests
 slackeventsapi
 sqlalchemy
 Flask==1.0.2
-Flask-pyoidc==1.3.0
+Flask-pyoidc
```

Comment:
Add version specifier for >=2.0.0.  Leaving it open means that version 1.3.0 is usable, even though it is not